RELEASING.md

```diff
@@ -1,9 +1,26 @@
-# Releasing
+# How to release v1
 
-1. Make changes.
-2. Determine next version. For the sake of this list, let's assume 1.2.3.
-3. Update `gax.go` with `const Version = "1.2.3"`. If v2, do the same to `v2/gax.go`.
-4. `git checkout -b my_branch && git commit && git push my_remote my_branch`
-5. Get it reviewed and merged.
-6. `git checkout master && git tag v1.2.3 && git push origin v1.2.3`
-7. Update the [releases page on GitHub](https://github.com/googleapis/gax-go/releases)
+1. Determine the current release version with `git tag -l`. It should look
+   something like `vX.Y.Z`. We'll call the current version `$CV` and the new
+   version `$NV`.
+1. On master, run `git log $CV..` to list all the changes since the last
+   release.
+   a. NOTE: Some commits may pertain to only v1 or v2. Manually introspect
+   each commit to figure which occurred in v1.
+1. Edit `CHANGES.md` to include a summary of the changes.
```

Comment:
Should these all be`1.`?

Reply:
Yeah. Github automagically changes all numbers in a list to be correctly ordered. So, if you write: 1, 7, 9 it'll show up as 1, 2, 3:

1. this is 1
7. this is 7
9. this is 9

(the counting starts with whatever the first number is: if you write 3, 7, 9 it'll show up as 3, 4, 5)